### DISCUSSION TITLE
How to set up cookies for Tiktok?

### DISCUSSION THREAD

**Discussion by linhcentrio** [AAndyProgram/SCrawler#113] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45778128/223315325-2c398b97-0f83-4576-854a-084e32caafba.png)
When you say "Copy all text with mouse.", i copied all text in that column and copy to your app but always error?
I can not understand this.

**Comment by AAndyProgram** [comment, marked as answer]
1. In addition to the value, cookies also include the name, domain and a few more parameters. So, you have to select all the text of all columns.
2. TikTok is not working right now.

**Comment by linhcentrio** [comment]
Thank you, but can you make a sample video "How to add cookies to your app"?
Or any Pictures?

**Reply by linhcentrio** [reply]
I gave up, it's too difficult to use. I've never used a software that difficult to use before.

**Reply by AAndyProgram** [reply]
Okay
